ranger-runner/src/test/java/io/smartcat/ranger/runner/LoadGeneratorRunnerTest.java

```diff
@@ -0,0 +1,32 @@
+package io.smartcat.ranger.runner;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.nio.file.Paths;
+
+import org.junit.Test;
+
+public class LoadGeneratorRunnerTest {
+
+    @Test(timeout = 3000)
```

Comment:
Relying on timeout proved to be shitty on travis. If tests start failing consider changing this to a concrete check.